.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,68 @@
+name: sriov-e2e-test
+run-name: sriov-e2e-test initiated by ${{ github.actor }}
+on:
+  pull_request:
+    types: [labeled]
+jobs:
+  e2e-tests:
+    runs-on: self-hosted
+    if: contains(github.event.label.name, 'e2e-test')
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v3
```

Comment:
We should use setup-python@v4 to avoid "deprecated" warnings on every run.

Reply:
I think updating setup-python version here is fine. I have another PR to update it in other actions. Python versions we can update later in a separate PR. There is no dependency on setup-python.